css/style.css

```diff
@@ -208,3 +208,9 @@ input.webform-calendar {
   float: left;
   margin: 0 20px 30px 0;
 }
+
+/* book pages */
+ul li.leaf {
```

Comment:
This wipes out list-style for all drupal lists but this pull request only fixes the list style for .block-menu-block

This should be moved to the only file and scoped to .block-menu-block